src/Boot/src/AbstractKernel.php

```diff
@@ -259,8 +264,12 @@ public function bootstrapped(Closure ...$callbacks): void
      * Add new dispatcher. This method must only be called before method `serve`
      * will be invoked.
      */
-    public function addDispatcher(DispatcherInterface $dispatcher): self
+    public function addDispatcher(string|DispatcherInterface $dispatcher): self
```

Comment:
Please describe the `$dispatcher` parameter

Reply:
Added